src/components/SortButton.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useState } from 'react'
+import { useSelector } from 'react-redux'
+import { css } from '../../styled-system/css'
+import theme from '../selectors/theme'
+import SortIcon from './icons/SortIcon'
+
+/**
+ * SortButton component.
+ * */
+const SortButton: React.FC<{ onSortChange: (sortOrder: 'alphabetical' | 'type') => void }> = ({ onSortChange }) => {
+  const isLightTheme = useSelector(state => theme(state) === 'Light')
+  const [isDropdownOpen, setDropdownOpen] = useState(false)
+  const [selectedSort, setSelectedSort] = useState<'alphabetical' | 'type'>('type')
+
+  const dropdownStyles = css({
+    opacity: 0,
+    visibility: 'hidden',
+    transition: 'opacity 0.3s ease-in-out, visibility 0.3s ease-in-out',
+    willChange: 'opacity, visibility',
+    position: 'absolute',
+    top: '100%',
+    right: 0,
+    backgroundColor: '{colors.bg}',
```

Comment:
This can just be `backgroundColor: 'bg'`, as PandaCSS knows the named colors from our config. 

The braces are only needed for interpolation.